packages/datadog-plugin-aws-sdk/src/base.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
```

Comment:
What is the reasoning behind the current file structure? I'm not sure it helps readability unless it would be split more specifically.

Reply:
updated per https://github.com/DataDog/dd-trace-js/pull/856#discussion_r381882631